Dockerfile

```diff
@@ -18,6 +18,14 @@ ENV READONLY_CACHE=true
 ###################
 FROM builder-cache-${use_cache} as builder
 
+# Information for Sentry Releases.
+ARG manual_sentry_release=false
+ARG is_default_branch=false
+ARG sentry_auth_token=''
+ENV MANUAL_SENTRY_RELEASE $manual_sentry_release
```

Comment:
Passing ARGs without additional ENVs meant that the data wasn't available on process.env. Which isn't quite what I was expecting based on what I was reading online. But I suppose it works, even if it's verbose.

Reply:
Interesting, TIL.